portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +26,23 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private ArrayList<String> names;
```

Comment:
From a Java perspective I'd probably prefer that you make this an immutable static list since it's never changed, something more like

```java
  private static final ImmutableList<String> NAMES = ImmutableList.of(
      "Cindy", "Anthony", "Charles", "Chris", "Ben");
```

To do that you will need to add a dependency on [Guava](https://guava.dev/).

This would also make your `init()` method unnecessary.